bower.json

```diff
@@ -22,7 +22,7 @@
   "homepage": "https://github.com/PolymerElements/paper-card",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
+    "polymer": "Polymer/polymer#^1.1.0",
```

Comment:
If someone already had polymer installed and this package and they wanted to get the latest version of this package they could get into the state where they would be using the 1.0.0 Release of polymer since it fits the semver constraints. Setting it to. 1.1.0 makes it so the lowest version that this package supports is that. Without this change I think some
People might get into some broken states. 

Thoughts?